shopfloor/models/stock_package_level.py

```diff
@@ -1,14 +1,20 @@
 from odoo import fields, models
 
+from .stock_move_line import StockMoveLine
+
 
 class StockPackageLevel(models.Model):
     _inherit = "stock.package_level"
 
-    shopfloor_postponed = fields.Boolean(
-        default=False,
+    # shopfloor_priority is set to this value when postponed
+    # consider it as the max value for priority
+    _SF_PRIORITY_POSTPONED = StockMoveLine._SF_PRIORITY_POSTPONED
```

Comment:
Does it still deserve a `shopfloor_postponed` compute+inverse field? Returning `True` if `shopfloor_priority == self._SF_PRIORITY_POSTPONED`, and setting `_SF_PRIORITY_POSTPONED` if we set `True`?
Like this scenarios and tests won't be aware of this internal constant.

Reply:
indeed. Also, this could be moved to a mixin

Reply:
pushed in a fixup

Reply:
BTW I also thought postponing could do `shopfloor_priority += 1` instead of an arbitrary value. Which means, if you postpone a bunch of line, and postpone again one of these lines when you reach the postponed lines, the line postponed 2 times will be at the end again.